guess.js

```diff
@@ -114,6 +120,8 @@ function guessCheck() {
 
     let successguess = true;
     for (let i = 1; i <= numbersOfLeters; i++) {
+        // [REVIEW] It would be nice if you can create a little function to generate the input ID ..
+        // .. this way you can avoid repeating the same code.
         const inputField = document.querySelector(`#guess-${currentTry}-input-${i}`)

```

Comment:
Define the function as 
```js
function makeGuessInputId(try, guessId) {
  return `guess-${try}-input-${guessId}`;
}
```
And then you will reuse this here and also [here](https://github.com/mohamed-sabry-1789/guess-word-game/blob/main/guess.js#L35C25-L35C46).

```suggestion
        // [REVIEW] It would be nice if you can create a little function to generate the input ID ..
        // .. this way you can avoid repeating the same code.
        const inputField = document.querySelector(`#${makeGuessInputId(currentTry, i)}`);

```

Reply:
iam still not understand  and why i cand do this i have allrady id and var

Reply:
It's just for reusability. 
There's a concept in software development called DRY (Don't Repeat Yourself), which basically means that if you find yourself writing the same code twice, this _usually_ means you should move this code to a function and reuse it.